src/OpenTK/Platform/Windows/WinGLNative.cs

```diff
@@ -1013,9 +1013,9 @@ private void ResetWindowState()
             suppress_resize--;
         }
 
-        private void GrabCursor(bool grab)
+        private void GrabCursor(bool shouldGrab)
```

Comment:
I'm not sure about this. Seems like it would be easier to use if it was just one method that took an on/off parameter vs having to do an if to select between two methods. That is compare:
```
GrabCursor(variable)
```
to
```
if (variable)
{
    GrabCursor();
}
else
{
    HideCursor();
}
```